src/gnc_environment.cpp

```diff
@@ -68,13 +68,25 @@ void sun_vector(double t, lin::Vector3f &s) {
   utl::rotate_frame(q_eci_perifocal_f, s);
 }
 
-void magnetic_field(double t, lin::Vector3f const &r, lin::Vector3f &b) {
+void sun_vector(double t, lin::Vector3d &s) {
```

Comment:
Same with this one

Reply:
No, because of the float `lin::Vector3f` overload just above it.